import org.apache.hadoop.fs.impl.WrappedIOException;
public class FunctionUse10646 {
public void funcUse() {
WrappedIOException wrappedioexception = new WrappedIOException();
wrappedioexception.getCause();
}
}